samples/buckets.js

```diff
@@ -67,6 +67,40 @@ async function listBuckets() {
   // [END storage_list_buckets]
 }
 
+async function getBucketMetadata(bucketName) {
```

Comment:
Could I trouble you to put this sample in its own file?  Then you can add metadata that gets picked up automatically.  See https://github.com/googleapis/nodejs-bigquery/blob/master/samples/getModel.js#L19 for an example!

Reply:
Sure thing! Thanks for the example.